src/BTCPayServer.Lightning.All/LightningConnectionString.cs

```diff
@@ -587,6 +622,9 @@ public override string ToString()
                 case LightningConnectionType.Ptarmigan:
                     builder.Append($";server={BaseUri}");
                     break;
+                case LightningConnectionType.LNbank:
+                    builder.Append($";server={BaseUri};api-token={ApiToken};wallet-id={WalletId}");
```

Comment:
why do you need a wallet id if you have an api token?

Reply:
You can have multiple wallets per token.

Reply:
Correct. With just the API key I wouldn‘t know which wallet to use. Having multiple wallets is a good way to separate stuff.